charts/milvus/templates/config.tpl

```diff
@@ -244,4 +244,31 @@ log:
     maxBackups: {{ .Values.log.file.maxBackups }}
   format: {{ .Values.log.format }}
 
-{{- end }}
+
+
+woodpecker:
```

Comment:
We won't add woodpecker configurations explicitly. User can add those configurations through `extraConfigFiles`.